docusaurus/docs/_cli-log-levels-and-formats.md

```diff
@@ -13,17 +13,17 @@ Output from Ziti components comes in three distinct styles. Choose the style of
 - `json` - a machine-readable format targeting automated processes for log aggregation/searching
 - `text` - a human-readable format using plain text (no ascii escape codes)
 
-```text
-ziti-router run ./router.yml --log-formatter pfxlog
+```text title="Run the router without colorized log levels"
+ziti router run config.yml --log-formatter text
```

Comment:
why does "router" show up under "controller" -- this doesn't fit here either. i think the whole section probably needs to be rearranged per the prior comment

Reply:
Now I see it was reused in two places... I think what this turns into is that it just doesn't make sense to reuse this content...

Reply:
Deleted this file and duplicated and tailored the content in controller and router deploy guides where it was being included as a partial.